otter/test/models/test_cass_models.py

```diff
@@ -3862,6 +3862,16 @@ def test_insert_empty(self):
             self.cache.insert_servers(self.dt, [], False),
             Effect(Constant(None)))
 
+    def test_insert_empty_delete(self):
+        """
+        `insert_servers` deletes servers when clear_others=True and does
+        nothing if passed list is empty
+        """
+        self.cache.delete_servers = lambda: Effect("delete")
+        eff = self.cache.insert_servers(self.dt, [], True)
+        self.assertEqual(eff.intent, "delete")
+        self.assertIsNone(resolve_effect(eff, None))
```

Comment:
I'm not sure what the point of this is. As far as I can tell, it's resolving the dummy effect that was set up in the test. Going further, I feel like the test could be written like this to get the same effect:

```
self.cache.delete_servers = lambda: 'delete_servers_effect'
self.assertEqual(self.cache.insert_servers(self.dt, [], clear_others=True), 'delete_servers_effect')
```